src/ray/object_manager/object_manager.cc

```diff
@@ -17,6 +17,7 @@ ObjectManager::ObjectManager(asio::io_service &main_service,
       // release_delay of 2 * config_.max_sends is to ensure the pool does not release
       // an object prematurely whenever we reach the maximum number of sends.
       buffer_pool_(config_.store_socket_name, config_.object_chunk_size,
+                   config_.max_sends,
```

Comment:
Can we use max_chunks to make it consistent with object_buffer_pool.h.

Reply:
The configuration `max_sends` is used in other places in the code where the name `max_sends` makes more sense than `max_chunks`. I think keeping it `max_sends` here makes more sense.